sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptionData.cs

```diff
@@ -1,59 +1,56 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Runtime.Serialization;
+using System.Text.Json;
 using Metadata = System.Collections.Generic.IDictionary<string, string>;
 
 namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
 {
     /// <summary>
     /// Represents the encryption data that is stored on the service.
     /// </summary>
-    [DataContract]
     public struct EncryptionData
     {
-
         /// <summary>
         /// The blob encryption mode.
         /// </summary>
-        [DataMember]
         public string EncryptionMode { get; set; }
 
         /// <summary>
-        /// A {@link WrappedKey} object that stores the wrapping algorithm, key identifier and the encrypted key.
+        /// A <see cref="WrappedKey"/> object that stores the wrapping algorithm, key identifier and the encrypted key.
         /// </summary>
-        [DataMember]
         public WrappedKey WrappedContentKey { get; set; }
 
         /// <summary>
         /// The encryption agent.
         /// </summary>
-        [DataMember]
         public EncryptionAgent EncryptionAgent { get; set; }
 
         /// <summary>
         /// The content encryption IV.
         /// </summary>
-        [IgnoreDataMember]
         public byte[] ContentEncryptionIV { get; set; }
 
-        /// <summary>
-        /// Base64-encoded content encryption IV.
-        /// </summary>
-        [DataMember(Name = "ContentEncryptionIV")]
-        public string Base64ContentEncryptionIV
-        {
-            get => Convert.ToBase64String(ContentEncryptionIV);
-            set => ContentEncryptionIV = Convert.FromBase64String(value);
-        }
-
 #pragma warning disable CA2227 // Collection properties should be read only
         /// <summary>
         /// Metadata for encryption. Currently used only for storing the encryption library, but may contain other data.
         /// </summary>
-        [DataMember]
         public Metadata KeyWrappingMetadata { get; set; }
 #pragma warning restore CA2227 // Collection properties should be read only
+
+        /// <summary>
+        /// Serializes this object to JSON.
+        /// </summary>
+        /// <returns></returns>
+        public string Serialize()
+            => JsonSerializer.Serialize(this);
+
+        /// <summary>
+        /// Deserializes an <see cref="EncryptionData"/> from JSON.
+        /// </summary>
+        /// <param name="json">JSON to deserialize.</param>
+        /// <returns></returns>
+        public static EncryptionData Deserialize(string json)
+            => JsonSerializer.Deserialize<EncryptionData>(json);
```

Comment:
We're not using the higher level `Deserialize`/`Serialize` types for performance and compat.  If you use this, this library won't run in an Azure Function (until they update the version of some of their components).

Reply:
Can you point me to a resource that demonstrates the methods you like?